pkg/shell/machine-dialogs.js

```diff
@@ -34,7 +34,7 @@ define([
     "data!shell/templates/sync-users.html",
     "data!shell/templates/unknown-hostkey.html",
     'shell/po',
-    "base1/patterns",
+    "./patterns",
```

Comment:
The actual name needs to go into require() calls, but ./ is a better way to refer to local stuff from define() calls. Given the data-main support is working now we should move away from most of the require() calls.